index.js

```diff
@@ -3,6 +3,14 @@ import { maybeDecode } from './decode.js'
 
 /** @typedef { 'Complete' | 'Partial' | 'Unknown' } DagStructure */
 
+/**
+ * @typedef {Object} Report
```

Comment:
today i learn
> You can use either object or Object on the first line.
– https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html#typedef-callback-and-param